app/validation/validator.py

```diff
@@ -128,13 +128,21 @@ def validate_questionnaire(self, json_to_validate):
 
     def _validate_section(self, section):
         errors = []
-        section_repeat = section.get("repeat", {})
+        section_repeat = section.get("repeat")
 
         if section_repeat:
             errors.extend(self._validate_list_exists(section_repeat["for_list"]))
             errors.extend(
                 self._validate_placeholder_object(section_repeat["title"], None)
             )
+
+        custom_section_summary = section.get("summary")
```

Comment:
Not a fan of custom_section_summary in the code and summary in the JSON summary.

Reply:
I went with `section_summary` as it is consistent with `section_repeat`